### PR TITLE
Include inline and persistent details when gathering xpaths

### PR DESCRIPTION
**Pull request by orangejenny**
https://manage.dimagi.com/default.asp?256148

@benrudolph / @sravfeyn

### CODE REVIEW

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/app_manager/suite_xml/post_process/instances.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def _get_all_xpaths_for_entry(self, entry):
         for datum in entry.datums:
             detail_ids.add(datum.detail_confirm)
             detail_ids.add(datum.detail_select)
+            detail_ids.add(datum.detail_inline)
+            detail_ids.add(datum.detail_persistent)
```
Comment (sravfeyn) [inline corehq/apps/app_manager/suite_xml/post_process/instances.py]: Would be good to have a test that reproduces the bug.

**Comment by sravfeyn** [comment]
Thanks Jenny!

File: corehq/apps/app_manager/suite_xml/post_process/instances.py (hunk 1 above)
Comment (sravfeyn) [inline corehq/apps/app_manager/suite_xml/post_process/instances.py]: Would be good to have a test that reproduces the bug.

Comment (orangejenny) [inline corehq/apps/app_manager/suite_xml/post_process/instances.py]: thanks for the nudge! https://github.com/dimagi/commcare-hq/pull/16713

**Pull request merged**